en/drafts/originals/text-mining-youtube-comments.md

```diff
@@ -479,22 +482,23 @@ wf_comment_plot <- ggplot(wf_comment_df) + geom_point(aes(x = theta, y = alpha,
+Based on this visualization, the channel's political affiliation does not seem to be a strong predictor of the commenters' political positions. When conducting your own research, you should update the partisan indicator described in the [Data Labeling section](#Data-Labeling) to match your own research needs, and ask yourself a similar set of questions.
 
-Based on this visualization, the political affiliation of the channels from which we gathered videos does not seem to be a strong predictor of the political positions of the people who leave comments.  When conducting your own research, you should update the partisan indicator described in the [Data Labeling section](#Data-Labeling) to match your own research needs, and ask yourself a similar set of questions.
+## Conclusion
 
-# Conclusion
 By this point of the lesson, you have downloaded a large corpus of YouTube video comments, processed them, analyzed them using the Wordfish model of text scaling, and produced several insightful visualizations. You can reuse the code in this lesson on your own YouTube comment dataset - to download the code for easy re-use, see the attached [R script](/ph-submissions/assets/text-mining-youtube-comments/youtube.R).
 
-If you used this lesson's data, based on these three visualizations you can tell that a broadly similar set of topics is discussed on left-leaning and right-leaning video comment threads on  YouTube videos focused on police brutality and questions about police funding. However, you have also seen an example of how to interpret these visualizations to learn more about what words describe the scale created by the Wordfish model, and also if all of your videos contributed equally to each pole of that scale (or not). 
+If you used this lesson's data, you can tell from these three visualizations that a broadly similar set of topics is discussed on comment threads for both left-leaning and right-leaning YouTube videos focused on police brutality and police funding. You have also seen that you can interpret these visualizations to discover which words determined the scale created by the Wordfish model, and which videos contributed to each pole of that scale. 
 
-These visualizations, and more granular analyses of the Wordfish model, will enable complex interpretations of textual meaning. That Wordfish can be useful for understanding the strange type of discourse that appears in YouTube comments is a fascinating revelation of its own.
+These visualizations, and more granular analyses of the Wordfish model, will enable complex interpretations of textual meaning. The fact that Wordfish can be useful for understanding the strange type of discourse that appears in YouTube comments is a fascinating revelation of its own.
 
-# Endnotes
+## Endnotes
 
 [^1]: It is not possible to fully cover the benefits and limitations of Wordfish in this lesson; see Nanni, et al. (2019) and this blog post by Jeff Antsen for more detail.

```

Comment:
Could you please provide a complete citation for Nanni, et al. (2019) and the blog post?